prog/pwm/fancontrol

```diff
@@ -67,6 +67,7 @@ function LoadConfig
 	FCFANS=`egrep '^FCFANS=.*$' $1 | sed -e 's/FCFANS=//g'`
 	MINPWM=`egrep '^MINPWM=.*$' $1 | sed -e 's/MINPWM=//g'`
 	MAXPWM=`egrep '^MAXPWM=.*$' $1 | sed -e 's/MAXPWM=//g'`
+	FCCHARCUR=$(grep -E '^FCCHARCUR=.*$' $1 | sed -e 's/FCCHARCUR=//g')
```

Comment:
So FCCHARCUR is the parameter ? Can we find something better that reflects what we are doing ? I have not checked later patches yet, but did you document the parameter in fancontrol.8 ?

Reply:
How about SQUAREPERFORMANCE   or in my opinion even better: SquarePerformance=0  or  1 switching it on and off?

Reply:
FCSQUARE, maybe, for "Fan Control Square" ? That would fit with the other parameter names. 0/1 for off/on sounds good.